apps/web/src/pages/projects/[project]/authentication/index.tsx

```diff
@@ -337,7 +371,7 @@ export default function ProjectAuthentication() {
                                 </FormItem>
                             )}
                             />
-                            <FormField
+                            {/* <FormField
```

Comment:
Is dit veld niet meer van toepassing?

Reply:
Dit veld had ik tijdelijk eruit gecomment, omdat hier de afbeelding uploader die jij ging bouwen terecht moest komen. Ik heb hem erin gelaten zodat we niet vergeten dat die hier toegepast moet worden.